web/src/components/netflow-table.tsx

```diff
@@ -94,9 +151,14 @@ const NetflowTable: React.FC<{
         columns={columns}
       />
       <Tbody>
-        {flows.map((f, i) => (
-          <NetflowTableRow key={i} flow={f} columns={columns} />
-        ))}
+        {error
+          ? errorState
+          : _.isEmpty(flows)
+          ? // empty flows might mean that page is loading or no flows found
+            loading
+            ? loadingContents
+            : noResultsFound
+          : flows.map((f, i) => <NetflowTableRow key={i} flow={f} columns={columns} />)}
```

Comment:
Maybe the nested conditions are a bit cumbersome but the npm formatter didn't allow me to organize it in another way 😥

Reply:
What about using a `let` that will be filled according to if / else conditions like : 
```
let bodyContent;
if(error) {
   bodyContent = (
    <Tr>
      <Td colSpan={columns.length}>
        <EmptyState data-test="error-state" variant={EmptyStateVariant.small}>
          <Title headingLevel="h2" size="lg">
            {t('Unable to get flows')}
          </Title>
          <EmptyStateBody>{error}</EmptyStateBody>
        </EmptyState>
      </Td>
    </Tr>
  );
} else if(_.isEmpty(flows)) {
   if(loading) {
      bodyContent = (
        <Tr>
          <Td colSpan={columns.length}>
            <Bullseye data-test="loading-contents">
              <Spinner size="xl" />
            </Bullseye>
          </Td>
        </Tr>
      );
   } else {
      bodyContent = (
        <Tr>
          <Td colSpan={columns.length}>
            <Bullseye data-test="no-results-found">
              <EmptyState variant={EmptyStateVariant.small}>
                <EmptyStateIcon icon={SearchIcon} />
                <Title headingLevel="h2" size="lg">
                  {t('No results found')}
                </Title>
                <EmptyStateBody>{t('Clear all filters and try again.')}</EmptyStateBody>
                {/*TODO: take onClick action after task NETOBSERV-74 is completed*/}
                <Button variant="link">{t('Clear all filters')}</Button>
              </EmptyState>
            </Bullseye>
          </Td>
        </Tr>
      );
   }
} else {
   bodyContent = flows.map((f, i) => <NetflowTableRow key={i} flow={f} columns={columns} />);
}
```

Then simply : 
`<Tbody>{bodyContent}</Tbody>` in the return statement

Reply:
Fine with me. I observed that, in the console code, if-elses were unused in favor of the ternary operator so I thought it was some kind on undesirable idiom.